### PR TITLE
PVW-6: Setting the project template

### PR DESCRIPTION
**Pull request by Julian-UBC**

### CODE REVIEW

**Review by mayaramadhina — APPROVED** [review]
logo looks good!

File: client/src/react-app-env.d.ts
Comment (filipomarcellino) [inline client/src/react-app-env.d.ts:1]: awesome you're already defining types for different files that will be imported

Comment (filipomarcellino) [reply client/src/react-app-env.d.ts:1]: gokillll

**Review by filipomarcellino — APPROVED** [review]
ok

**Pull request merged**